pkg/controllers/provisioningv2/machineconfigcleanup/controller.go

```diff
@@ -179,6 +179,10 @@ func cleanupObjects(token string) []runtime.Object {
 								{
 									Name:  fmt.Sprintf("%s-pod", prefix),
 									Image: image,
+									SecurityContext: &corev1.SecurityContext{
+										RunAsUser:  &[]int64{1000}[0],
+										RunAsGroup: &[]int64{1000}[0],
```

Comment:
As per off-line chat, we could run this in the context of `nobody:nobody`:
```suggestion
										RunAsUser:  &[]int64{65534}[0],
										RunAsGroup: &[]int64{65534}[0],
```